src/org/rascalmpl/library/util/PathConfig.java

```diff
@@ -650,83 +654,46 @@ private static IList getPomXmlCompilerClasspath(ISourceLocation manifestRoot) {
                 return vf.list();
             }
 
-            String mvnCommand = computeMavenCommandName();
-
-            installNecessaryMavenPlugins(mvnCommand);
-
-            // Note how we try to do this "offline" using the "-o" flag
-            ProcessBuilder processBuilder = new ProcessBuilder(mvnCommand, 
-                "--batch-mode", 
-                "-o", 
-                "dependency:build-classpath",
-                "-DincludeScope=compile",
-                trustStoreFix()
-            );
-
-            processBuilder.directory(new File(manifestRoot.getPath()));
-            processBuilder.environment().put("JAVA_HOME", System.getProperty("java.home", System.getenv("JAVA_HOME")));
-
-            Process process = processBuilder.start();
+            var maven = new MavenCli();
+            var tempFile = Files.createTempFile("rascal-classpath-", ".tmp");
+            
+            maven.doMain(buildRequest(new String[] {"-quiet", "-o", "dependency:build-classpath", "-DincludeScope=compile", "-Dmdep.outputFile=" + tempFile.toString()}, manifestRoot));
```

Comment:
What does the switch from `--batch-mode` to `--quit` do?

Reply:
`batch-mode` suppressed color output, `quiet` suppressed (non-error) output altogether. Currently, all output is suppressed